examples/TESSLightCurves.ipynb

```diff
@@ -0,0 +1,279 @@
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a3e9c2-e9e3-48ca-a21c-0ae64bb549f6",
+   "metadata": {},
+   "source": [
+    "**Now, we load some test data!**\n",
+    "\n",
+    "We use simulated subjects from the PH-TESS dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbf7b1b5-79b7-4a2a-a8e9-15f196a15ca6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "s94812564 = 'https://panoptes-uploads.zooniverse.org/subject_location/c4a2a54a-da3b-4f1d-9823-702549a2806a.txt'\n",
+    "s94880899 = 'https://panoptes-uploads.zooniverse.org/subject_location/03caefe6-7d48-4225-9cc0-e5a60f337af8.txt'\n",
+    "s94880882 = 'https://panoptes-uploads.zooniverse.org/subject_location/7578bd5d-b8da-4c42-8da7-5f897ded018e.txt'\n",
+    "response = json.loads(requests.get(s94812564).text)\n",
+    "days = np.array(response['x'])\n",
+    "flux = np.array(response['y'])\n",
+    "normalised_days = lambda d: d / days.max()\n",
+    "x = np.array(list(map(normalised_days, days)))\n",
+    "normalised_flux = lambda f: ((f - flux.min()) / (flux.max() - flux.min()))\n",
+    "y = np.array(list(map(normalised_flux, flux)))\n",
+    "\n",
+    "\n",
+    "plt.plot(days,y)\n",
+    "plt.ylabel('Normalised Flux')\n",
+    "plt.xlabel('Days')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0f6edf6-59bf-4bdc-a9e7-c7f097cd08d3",
+   "metadata": {},
+   "source": [
+    "**Set up some universal sonification parameters and classes for the examples below**\n",
+    "\n",
+    "For all examples we use the `Synthesizer` generator to create a 20 second, stereo sonification."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "009d6c9a-fb4c-4cd9-a87d-ccae4f280d89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# specify audio system (e.g. mono, stereo, 5.1, ...)\n",
+    "system = \"stereo\"\n",
+    "\n",
+    "# length of the sonification in s\n",
+    "length = 20.\n",
+    "\n",
+    "# set up synth and turn on LP filter\n",
+    "generator = Synthesizer()\n",
+    "generator.load_preset('pitch_mapper')\n",
+    "generator.preset_details('pitch_mapper')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7a7b7ed-6da8-4694-8aef-df00a7277d42",
+   "metadata": {},
+   "source": [
+    "### <u>Example 1</u> &nbsp; **Pitch Mapping**\n",
+    "\n",
+    "Set `azimuth` to pan from left to right along the x-axis. Set `pitch_shift` to map dips in flux (`y`) as increases in pitch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c92480b-345a-4212-883c-b9eded4060d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "notes = [[\"A2\"]]\n",
+    "score =  Score(notes, length)\n",
+    "\n",
+    "data = {'pitch':1.,\n",
+    "        'time_evo':x,\n",
+    "        'azimuth':(x*0.5+0.25) % 1,\n",
+    "        'polar':0.5,\n",
+    "        'pitch_shift':(y*10+1)**-0.7}\n",
```

Comment:
The `**-0.7` here means that higher flux => lower pitch - could just map `'pitch_shift':y` here instead to get high pitch => high flux?